Resources/Textures/Corvax/Objects/Misc/envelope.rsi/meta.json

```
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "size": {
```

Забыл скинуть копирайты
```suggestion
  "version": 1,
  "license": "CC-BY-SA-3.0",
  "copyright": "Taken from infinitystation at commit https://github.com/infinitystation/Baystation12/commit/034c0be53463a2f889d052e04c688c3ef2acc956",
  "size": {
```